tests/test_integration.py

```diff
@@ -32,6 +32,12 @@
 from pex.util import DistributionHelper, named_temporary_file
 
 
+def make_env(**kwargs):
+  env = os.environ.copy()
+  env.update((k, str(v)) for k, v in kwargs.items())
```

Comment:
This works fine for Py2 vs Py3 interop? `str` in Py2 will have `v` be bytes, but Py3 unicode.

I see it's test code so probably fine.

Reply:
Wish we had `future` library in PEX. I know we discussed not spending time rewriting Py2 vs Py3 compatibility to use the Py3-first approach, but perhaps it would be worth installing to allow new and refactored code to use the `future` backports.

Reply:
> This works fine for Py2 vs Py3 interop?